Gallery.Api/Services/UserArticleService.cs

```diff
@@ -478,6 +495,60 @@ private async Task<UnreadArticles> GetUnreadArticlesAsync(Guid exhibitId, Guid u
 
         }
 
+        public async Task<bool> LogXApiAsync(Uri verb, Article article, Guid exhibitId, CancellationToken ct)
```

Comment:
This seems to be largely or entirely duplicated from ArticleService. Maybe these could be consolidated in the XApiService?